AGENTS.md

```
@@ -0,0 +1,316 @@
+# Create a package tarball for local testing
+npx nx pack @analog-tools/auth
+
+# Release workflow (versions and publishes)
+npx nx release
+```
+
+## Architecture
+
+### Package Relationships
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)